sdk/servicebus/azure-messaging-servicebus/src/samples/README.md

```diff
@@ -30,18 +30,24 @@ connection string value can be obtained by:
 
 ## Examples
 
+### Asynchronously sending and receiving
+
 - [Send messages][SendMessageAsyncSample]
 - [Send messages using Azure Identity][SendMessageWithAzureIdentityAsyncSample]
-- [Send message batches synchronously][SendMessageBatchSyncSample]
-- [Schedule and cancel a message][SendScheduledMessageAndCancelAsyncSample]
-- [Receive messages synchronously][ReceiveMessageSyncSample]
+- [Process all messages using processor][ServiceBusProcessorSample]
 - [Receive and auto-complete messages][ReceiveMessageAsyncSample]
-- [Receive messages using Azure Identity][ReceiveMessageAzureIdentityAsyncSample]
-- [Receive and settle messages via complete or abandon][ReceiveMessageAndSettleAsyncSample]
+- [Receive and manually settle messages][ReceiveMessageAndSettleAsyncSample]
+- [Receive messages with auto-lock renewal][ReceiveMessageAutoLockRenewal]
+- [Schedule and cancel a message][SendScheduledMessageAndCancelAsyncSample]
 - [Peek at a message][PeekMessageAsyncSample]
 
+### Synchronous sending and receiving
+- [Send message batches synchronously][SendMessageBatchSyncSample]
+- [Receive messages synchronously][ReceiveMessageSample]
+
 ### Message sessions
-- [Send and receive messages from a session][SendAndReceiveSessionMessageSample]
+- [Send messages to a session][SendSessionMessageSample]
```

Comment:
Link to this shows me 404 : "Send messages to a session"

Reply:
These 404 links are because its referencing "master". Once this is merged, it'll work.